django_app/redbox_app/redbox_core/models.py

```diff
@@ -445,6 +445,7 @@ def get_ordered_by_citation_priority(cls, chat_message_id: uuid.UUID) -> Sequenc
 class Chat(UUIDPrimaryKeyBase, TimeStampedModel, AbstractAISettings):
     name = models.TextField(max_length=1024, null=False, blank=False)
     user = models.ForeignKey(User, on_delete=models.CASCADE)
+    archived = models.BooleanField(default=False)
```

Comment:
```suggestion
    archived = models.BooleanField(default=False, null=True, blank=True)
```

you need to add ` null=True, blank=True` when adding new fields to existing models as the default wont back populate values